Integrations/integration-ATD.yml

```diff
@@ -37,672 +38,670 @@ configuration:
+    def check_task_status_by_jobId(jobIds):
+        taskIds = []
+        for i in range(len(jobIds)):
+            taskIds.append((http_request('php/getTaskIdList.php?jobId='+jobIds[i],'get', API_HEADERS))['result']['taskIdList'])
+        return check_task_status_by_taskId(taskIds)
+
+    def check_task_status_command():
+        ids = []
+        result = {}
+        args = demisto.args()
+
+        if ( ('jobId' not in args and 'taskId' not in args) or ('jobId' in args and 'taskId' in args) ):
+            return_error('You must specify one (and only one) of the following: jobId, taskId.')
+
+        if ('jobId' in args):
+            ids = string_arr_to_int_arr(argToList(args['jobId']))
+            result = check_task_status_by_jobId(ids)
+
+        elif ('taskId' in args):
+            ids = string_arr_to_int_arr(argToList(args['taskId']))
+            result = check_task_status_by_taskId(ids)
+
+        md = tableToMarkdown(
+            'ATD Sandbox Task Status',
+            result['tasks'],
+            (result['tasks'][0]).keys()
+        )
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': result['multipleResults'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': {
+                  "ATD.status": result['status'], #backward compatability
+                  "ATD.Task(val.taskId==obj.taskId)": result['tasks']
             }
-        };
-    }
-
-    function prettifyListProfilesRes(profiles){
-        var prettyProfiles = [];
-        for (i = 0; i < profiles.length; i++){
-            prettyProfiles[i] = {
-                'Name': profiles[i].name,
-                'AnalyzerProfileId': profiles[i].vmProfileid,
-                'Description': profiles[i].vmDesc,
-                'Sandbox': profiles[i].sandbox === 1 ? 'True' : 'False',
-                'Internet': profiles[i].internet === 1 ? 'True' : 'False',
-                'LocalBlackList': profiles[i].locBlackList === 1 ? 'True' : 'False'
+        })
+
+    def get_taskIds(jobIds):
+        results = []
+        res = {}
+        for i in range(len(jobIds)):
+            res = http_request('php/getTaskIdList.php?jobId=' + str(jobIds[i]),'get', API_HEADERS)
+            results.append(res)
+        return results
+
+    def get_taskIds_command():
+        jobIds = string_arr_to_int_arr(argToList(demisto.args()['jobId']))
+        results = get_taskIds(jobIds)
+
+        multipleMd = []
+        ec = []
+        for i in range(len(results)):
+            multipleMd.append({
+                'taskId': results[i]['result']['taskIdList'],
+                'jobId': jobIds[i]
+            })
+            ec.append({
+                'taskId': results[i]['result']['taskIdList'],
+                'jobId': jobIds[i]
+            })
+
+        md = tblToMd('ATD TaskIds and JobIds List', multipleMd, ['taskId', 'jobId'])
```

Comment:
Please use `tableToMarkdown` (easier for people who are not familiar with our code to understand)